src/main/java/com/aws/greengrass/device/ClientDevicesAuthService.java

```diff
@@ -99,9 +97,13 @@ public ClientDevicesAuthService(Topics topics, GroupManager groupManager, Certif
     protected void install() throws InterruptedException {
         super.install();
         //handleConfiguration
-        this.deviceGroupsTopics = this.config.lookupTopics(CONFIGURATION_CONFIG_KEY, DEVICE_GROUPS_TOPICS);
-        this.deviceGroupsTopics.subscribe(this::handleConfigurationChange);
-        this.config.lookup(CONFIGURATION_CONFIG_KEY, CA_TYPE_TOPIC).subscribe(this::updateCAType);
+        this.config.lookupTopics(CONFIGURATION_CONFIG_KEY).subscribe((whatHappened, node) -> {
```

Comment:
ignore whathappened's that you don't care about. Specifically `interiorAdded` and `timestampUpdated`

Reply:
Addressed. Thanks!